lib/stdlib/test/binary_module_SUITE.erl

```diff
@@ -1373,3 +1376,38 @@ check_no_invalid_read_bug(I) ->
     binary:encode_unsigned(N+N),
     binary:encode_unsigned(N+N, little),
     check_no_invalid_read_bug(I+1).
+
+secure_compare(Config) when is_list(Config) ->
+    true  = binary:secure_compare(<<"">>, <<"">>),
```

Comment:
Could add some tests that check against random inputs.

Reply:
Could also add some tests to ensure that time to return is approximately the for two arguments regardless of their order, may be brittle.